deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -128,6 +128,30 @@ def search(
     ):
         """DeepLakeVectorStore search method that combines embedding search, metadata search, and custom TQL search.
 
+        Examples:
+            >>> # Search using an embedding function and prompt
+            >>> data = vector_store.search(
+            >>>        prompt = "What does this chatbot do?",
```

Comment:
we should not use `prompt` variable, the context for vector search is different

Reply:
I understand you use `query` for TQL, but `prompt` is pretty unintuitive

Reply:
`prompt` is an input to LLM to condition it on some output, not to do search. 

Since `query` is overused what about `searchQuery`?